### PR TITLE
Add docs for passing masterKey to user._linkWith

### PR DESCRIPTION
**Pull request by acinader**

### CODE REVIEW

**Comment by acinader** [comment]
@Tom-Fox1 fyi

File: _includes/js/users.md [hunk 1]
```diff
@@ -474,25 +474,49 @@ The body of the response will contain the `objectId`, `createdAt`, `sessionToken
...
+
 ### Unlinking
 
 Unlinking an existing user with a service also uses \_linkWith() method to clear `authData` from the user by setting the `authData` for the service to `null`.  For example, unlinking a user with a Facebook account would use a request like this:
 
 ```javascript
-let user = new Parse.User();
+const user = new Parse.User();
 user.id = "uMz0YZeAqc";
 user._linkWith("facebook", {}).then(function(user){
```
Comment (dplewis) [inline _includes/js/users.md]: `user._linkWith("facebook", {})` I don't think this works.

should be replaced with `_unlinkFrom` or `user._linkWith("facebook", { authData: null })`

File: _includes/js/users.md [hunk 2]
```diff
@@ -474,25 +474,49 @@ The body of the response will contain the `objectId`, `createdAt`, `sessionToken
...
+  'CustomAdapter',
+  myAuthData,
+  { useMasterKey: true }
+);
+```
+
+On rest, web, mobile, or TV clients, users can then login using the `CustomAdapter` by passing `myAuthData`:
+
+```javascript
+const loggedIn = await Parse.User.logInWith('CustomAdapter', myAuthData);
```
Comment (dplewis) [inline _includes/js/users.md:510]: An example of CustomAdapter, server config and `Parse.User._registerAuthenticationProvider`  should be included.

**Review by dplewis — CHANGES_REQUESTED** [review]

**Review by TomWFox — APPROVED** [review]
I fixed the link to the 3rd party auth section in the server guide.

I can't comment on the code but the wording looks good to me!

**Comment by dplewis** [comment]
Good catch!

**Pull request merged**